pcstac/pcstac/main.py

```diff
@@ -63,11 +77,28 @@
     ),
     client=PCClient.create(post_request_model=search_post_request_model),
     extensions=EXTENSIONS,
-    app=FastAPI(root_path=APP_ROOT_PATH, default_response_class=ORJSONResponse),
+    app=FastAPI(
+        root_path=APP_ROOT_PATH,
+        default_response_class=ORJSONResponse,
+        lifespan=lifespan,
+    ),
     search_get_request_model=search_get_request_model,
     search_post_request_model=search_post_request_model,
     response_class=ORJSONResponse,
     exceptions={**DEFAULT_STATUS_CODES, **PC_DEFAULT_STATUS_CODES},
+    middleware=[
+        Middleware(BrotliMiddleware),
+        Middleware(ProxyHeaderMiddleware),
```

Comment:
both are defined in the default stac-fastapi so they should have been present in the application before